Makefile

```diff
@@ -15,6 +15,36 @@ ut: update-version
 	calico/test \
 	nosetests tests/unit -c nose.cfg
 
+# Run system tests.
+st: docker-image run-etcd run-k8s-apiserver
```

Comment:
Could you add this as a dependency of the `ci` target so that it gets run on Semaphore?

e.g https://github.com/projectcalico/k8s-policy/pull/86/files